### PR TITLE
Remove python version and odoo release

### PR DESCRIPTION
**Pull request by konykon**

### CODE REVIEW

**Review by oyale — APPROVED** [review]

**Comment by enricostano** [comment]
Can we add some documentation with some example when and how to use these variables?

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ This an Ansible role for provisioning Odoo. It supports:
 * Odoo 11
 * Odoo 10
 
-I has not been tested yet with Odoo 13.
+It has not been tested yet newer Odoo versions.
```
Comment (enricostano) [inline README.md:9]:
```suggestion
It has not been tested yet with newer Odoo versions.
```

File: README.md [hunk 2]
```diff
@@ -125,6 +125,14 @@ In order to force update an odoo module or a list of modules execute provisionin
...
 ```
 
+* Remove python version and odoo release
+
+In order to update python version and odoo release we remove the installed versions first defining on the inventory the ansible variables that are listed below as true.
+```yml
+remove_python_version: true
+remove_odoo_release: true
+```
+
```
Comment (enricostano) [inline README.md:135]: Should we remove or set to `false` these variables afterwards?

**Comment by danypr92** [comment]
Why do we need to remove Python before installing the new interpreter?

We use pyenv to maintain different versions of Python in the same server when we are trying to upgrade the Python version.

With this process we remove the Python before install the new one, and if the compilation of the new Python fails, we need to compile again the removed Python version.

Can we remove the unused Python version once the new version is working?

**Pull request closed without merging**